src/js/index.js

```diff
@@ -317,8 +317,9 @@ export default class bulmaCalendar extends EventEmitter {
     this.timePicker.value(new Date());
     this.timePicker.refresh();
 
-    // TODO: check if closeOnSelect
-    this.save();
+    if (this.options.closeOnSelect && this.options.displayMode !== 'inline') {
+      this.save();
+    }
```

Comment:
Can you explain why this change was made? Moving the `save()` and `emit()` calls inside the conditional prevents the `select` event from firing in most cases.